Explorer/Assets/DCL/ExplorePanel/ExplorePanelController.cs

```diff
@@ -59,6 +66,16 @@ public class ExplorePanelController : ControllerBase<ExplorePanelView, ExplorePa
             this.profileWidgetController = profileWidgetController;
             this.systemMenuController = systemMenuController;
             this.dclInput = dclInput;
+            this.notificationBusController = notificationBusController;
+            this.mvcManager = mvcManager;
+            this.notificationBusController.SubscribeToNotificationType(NotificationType.REWARD_ASSIGNMENT, OnEventTriggered);
+        }
+
+        private void OnEventTriggered(object[] parameters)

```

Comment:
Let's make more verbose. How about `OnRewardAssigned` or at least something that will highlight it relation to `Notification`?